extensions/objects/templates/custom-osk-letters-editor.component.html

```diff
@@ -16,75 +16,75 @@
   <div class="oppia-custom-letters">
     <div *ngIf="currentTab === latinTab && lettersAreLowercase" class="oppia-custom-letters-latin-lower">
       <div class="oppia-custom-letters-row-1">
-        <button id="custom-letter-{{ letter }}" [attr.class]="(value.indexOf(letter) === -1) ? 'oppia-custom-letters-inactive' : 'oppia-custom-letters-active'" *ngFor="let letter of latinLowerCase[0]" (click)="updateLettersList(letter)">
+        <button id="custom-letter-{{ letter }}" [attr.class]="(value?.indexOf(letter) === -1) ? 'oppia-custom-letters-inactive' : 'oppia-custom-letters-active'" *ngFor="let letter of latinLowerCase[0]" (click)="updateLettersList(letter)">
```

Comment:
Why is the '?' needed?

Reply:
Using `?` avoids the `indexOf` function call on `value` when it is undefined or null. So, this prevents console errors.